### ISSUE TITLE
feat : full user profile picture 

### ISSUE THREAD

**Issue by RishiSuthar** [SoCuul/SCInsta#75]
Allow to view the full profile picture of a user (square) just like in IG rocket when long click on profile picture. Instead of downloading

**Comment by SoCuul** [comment]
Do you mean something like this?
![](https://github.com/SoCuul/SCInsta/assets/63339559/67336904-ba6b-4640-ba5c-50c4dc60bab2)

**Comment by RishiSuthar** [comment]
> Do you mean something like this? ![](https://private-user-images.githubusercontent.com/63339559/344436472-67336904-ba6b-4640-ba5c-50c4dc60bab2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk3MDc1OTUsIm5iZiI6MTcxOTcwNzI5NSwicGF0aCI6Ii82MzMzOTU1OS8zNDQ0MzY0NzItNjczMzY5MDQtYmE2Yi00NjQwLWJhNWMtNTBjNGRjNjBiYWIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjMwVDAwMjgxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFlNzE0ZjQxNDI1ZWRiM2U1ZmFlYTNjM2VjZjlkZjBiOTZjOWVmYjU4MWNhYWU0MDEwNmQ3YmFmYjBjMjI3MTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QxJ3TfxGeWckG2jfkyrsB6_q_-LcIvbiy-rHIZQiJGc)

Yeah somewhat like that, but it should take the whole page. Basically you’re extracting the profile image

**Comment by SoCuul** [comment]
> > Do you mean something like this? ![](https://private-user-images.githubusercontent.com/63339559/344436472-67336904-ba6b-4640-ba5c-50c4dc60bab2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk3MDc1OTUsIm5iZiI6MTcxOTcwNzI5NSwicGF0aCI6Ii82MzMzOTU1OS8zNDQ0MzY0NzItNjczMzY5MDQtYmE2Yi00NjQwLWJhNWMtNTBjNGRjNjBiYWIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjMwVDAwMjgxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFlNzE0ZjQxNDI1ZWRiM2U1ZmFlYTNjM2VjZjlkZjBiOTZjOWVmYjU4MWNhYWU0MDEwNmQ3YmFmYjBjMjI3MTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QxJ3TfxGeWckG2jfkyrsB6_q_-LcIvbiy-rHIZQiJGc)
> 
> Yeah somewhat like that, but it should take the whole page. Basically you’re extracting the profile image

So essentially you want the quick look preview for the image. Eg. 
<img width="220" alt="" src="https://github.com/SoCuul/SCInsta/assets/63339559/9a7dd32b-9d5a-460f-96ee-041435ae0225">

**Comment by RishiSuthar** [comment]
> > > Do you mean something like this? ![](https://private-user-images.githubusercontent.com/63339559/344436472-67336904-ba6b-4640-ba5c-50c4dc60bab2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk3MDc1OTUsIm5iZiI6MTcxOTcwNzI5NSwicGF0aCI6Ii82MzMzOTU1OS8zNDQ0MzY0NzItNjczMzY5MDQtYmE2Yi00NjQwLWJhNWMtNTBjNGRjNjBiYWIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjMwVDAwMjgxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFlNzE0ZjQxNDI1ZWRiM2U1ZmFlYTNjM2VjZjlkZjBiOTZjOWVmYjU4MWNhYWU0MDEwNmQ3YmFmYjBjMjI3MTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QxJ3TfxGeWckG2jfkyrsB6_q_-LcIvbiy-rHIZQiJGc)
> > 
> > 
> > Yeah somewhat like that, but it should take the whole page. Basically you’re extracting the profile image
> 
> So essentially you want the quick look preview for the image. Eg. <img alt="" width="220" src="https://private-user-images.githubusercontent.com/63339559/344437677-9a7dd32b-9d5a-460f-96ee-041435ae0225.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk3MTA4NzksIm5iZiI6MTcxOTcxMDU3OSwicGF0aCI6Ii82MzMzOTU1OS8zNDQ0Mzc2NzctOWE3ZGQzMmItOWQ1YS00NjBmLTk2ZWUtMDQxNDM1YWUwMjI1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjMwVDAxMjI1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE5NGI1NDJmZTM1M2ZkZDkwOWQ4MzQ2ZjdjYjljNzRkNTU2N2EzMzUyMWYwNzVhMWExZGQ3OTczOWRlMDQ0ODcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Izfld-cNizjlQHzkvohkuVY8yjT2lBYloUfjPwyuxNY">

Yeah exactly, not necessary to put the sharing option. Mainly for viewing. Would be cool 

People can screenshot if they need 😂😅

**Comment by SoCuul** [comment]
The sharing option is just built-in to the ios quick look feature itself!
Which also provides a way to download the profile picture, if people so choose.

**Comment by RishiSuthar** [comment]
> The sharing option is just built-in to the ios quick look feature itself! Which also provides a way to download the profile picture, if people so choose.

Perfect, would be an amazing integration😇

**Comment by SoCuul** [comment]
👀 

https://github.com/user-attachments/assets/42393070-f27a-4cbf-a70d-94b7e406da6a

**Comment by SoCuul** [comment]
Available in the latest version! 😁
